lib/Less/Functions.php

```diff
@@ -813,11 +813,11 @@ private function _isa($n, $type) {
 		return is_a($n, $type) ? new Less_Tree_Keyword('true') : new Less_Tree_Keyword('false');
 	}
 
-	public function tint($color, $amount) {
+	public function tint($color, $amount = null) {
```

Comment:
Ok, so since this is a port of `less.js`, we should do it as they did. I removed my second commit and will create a new pull request.